info-v4-resources.go

```diff
@@ -34,34 +36,42 @@ import (
 
 // PaginatedPoolsResponse represents a paginated response for pools
 type PaginatedPoolsResponse struct {
-	Results []PoolResource `json:"results" msg:"r,omitempty"`
-	Count   int            `json:"count" msg:"c"`
-	Total   int            `json:"total" msg:"t"`
-	Offset  int            `json:"offset" msg:"o"`
+	Results      []PoolResource `json:"results" msg:"r,omitempty"`
+	Count        int            `json:"count" msg:"c"`
+	Total        int            `json:"total" msg:"t"`
+	Offset       int            `json:"offset" msg:"o"`
+	Sort         string         `json:"sort" msg:"s"`
+	SortReversed bool           `json:"sortReversed" msg:"sr"`
 }
 
 // PaginatedNodesResponse represents a paginated response for nodes
 type PaginatedNodesResponse struct {
-	Results []NodeResource `json:"results" msg:"r,omitempty"`
-	Count   int            `json:"count" msg:"c"`
-	Total   int            `json:"total" msg:"t"`
-	Offset  int            `json:"offset" msg:"o"`
+	Results      []NodeResource `json:"results" msg:"r,omitempty"`
+	Count        int            `json:"count" msg:"c"`
+	Total        int            `json:"total" msg:"t"`
+	Offset       int            `json:"offset" msg:"o"`
+	Sort         string         `json:"sort" msg:"s"`
+	SortReversed bool           `json:"sortReversed" msg:"sr"`
 }
 
 // PaginatedDrivesResponse represents a paginated response for drives
 type PaginatedDrivesResponse struct {
-	Results []DriveResource `json:"results" msg:"r,omitempty"`
-	Count   int             `json:"count" msg:"c"`
-	Total   int             `json:"total" msg:"t"`
-	Offset  int             `json:"offset" msg:"o"`
+	Results      []DriveResource `json:"results" msg:"r,omitempty"`
+	Count        int             `json:"count" msg:"c"`
+	Total        int             `json:"total" msg:"t"`
+	Offset       int             `json:"offset" msg:"o"`
+	Sort         string          `json:"sort" msg:"s"`
+	SortReversed bool            `json:"sortReversed" msg:"sr"`
 }
 
 // PaginatedErasureSetsResponse represents a paginated response for erasure sets
 type PaginatedErasureSetsResponse struct {
-	Results []ErasureSetResource `json:"results" msg:"r,omitempty"`
-	Count   int                  `json:"count" msg:"c"`
-	Total   int                  `json:"total" msg:"t"`
-	Offset  int                  `json:"offset" msg:"o"`
+	Results      []ErasureSetResource `json:"results" msg:"r,omitempty"`
+	Count        int                  `json:"count" msg:"c"`
+	Total        int                  `json:"total" msg:"t"`
+	Offset       int                  `json:"offset" msg:"o"`
+	Sort         string               `json:"sort" msg:"s"`
```

Comment:
document what all values `Sort` might contain, looks like its a free-form string if it has a defined type then please define them as constants